packages/sentry-client/src/sentry/components/Status.tsx

```diff
@@ -8,27 +8,33 @@ interface IStatusProps {
 
 export class Status extends React.Component<IStatusProps, any> {
 	render() {
-		let className, style;
+		let className, color;
```

Comment:
This could stand to be in a `getStyles` type function. The JSX elements clean up `render` so much in the `Server` component, so I figure you could do the same with the styles here to similarly clean up this `render`